app/assets/stylesheets/helpers/_core.scss

```diff
@@ -298,38 +298,6 @@ h4 {
   }
 }
 
-#wrapper.businesslink #legacy-sources,
-#wrapper.directgov #legacy-sources {
-  display: block;
-}
-
-#wrapper.businesslink #legacy-sources li.businesslink {
-	background-image: image-url("businesslink-logo.png");
```

Comment:
Yep don't worry about making these changes, performanceplatform-selfserve won't be updated and is only a prototype